### PR TITLE
🎨 Sort imports automatically with Prettier plugin

### PR DESCRIPTION
**Pull request by max-programming**

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -14,7 +14,12 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react-jsx"
+    "jsx": "react-jsx",
+    "baseUrl": ".",
+    "paths": {
+      "@/*": ["./src/*"],
```
Comment (igitur) [inline tsconfig.json:20]: This config doesn't seem to be working. Therefore the tests fail. Also, if you can, please split up this change into a separate commit, as it's not directly related to the prettier plugin change.

File: src/App.tsx [hunk 2]
```diff
@@ -251,13 +255,13 @@ function App() {
...
   return (
-    <div className="h-screen flex flex-col">
+    <div className="flex flex-col h-screen">
       <Navbar
         setIsInfoModalOpen={setIsInfoModalOpen}
         setIsStatsModalOpen={setIsStatsModalOpen}
         setIsSettingsModalOpen={setIsSettingsModalOpen}
       />
-      <div className="pt-2 px-1 pb-8 md:max-w-7xl w-full mx-auto sm:px-6 lg:px-8 flex flex-col grow">
+      <div className="flex flex-col w-full px-1 pt-2 pb-8 mx-auto md:max-w-7xl sm:px-6 lg:px-8 grow">
```
Comment (igitur) [inline src/App.tsx:264]: Are these changes done by the prettier plugin?

Comment (max-programming) [reply src/App.tsx:264]: Unfortunately these might be done by a VS Code extension I have called Headwind. I'll disable that and commit again

File: tsconfig.json (hunk 1 above)
Comment (max-programming) [reply tsconfig.json:20]: Sure. But can you show me the vercel logs in some way?

File: src/App.tsx (hunk 2 above)
Comment (igitur) [reply src/App.tsx:264]: The link to the build logs is automatically attached to this PR, but here it is:

https://github.com/cwackerfuss/react-wordle/runs/6226460243?check_suite_focus=true

File: tsconfig.json (hunk 1 above)
Comment (igitur) [reply tsconfig.json:20]: The link to the build logs is automatically attached to this PR, but here it is:

https://github.com/cwackerfuss/react-wordle/runs/6226460243?check_suite_focus=true

I get the same when I do `npm run test` on my own machine.

File: src/App.tsx (hunk 2 above)
Comment (igitur) [reply src/App.tsx:264]: I see there is a prettier plugin available to sort TailwindCSS classes. Maybe that's a good idea, but as a separate commit or PR.

File: tsconfig.json (hunk 1 above)
Comment (max-programming) [reply tsconfig.json:20]: Fixed it at a501aea421d35245e4644611680511c43a2e27f7

File: src/App.tsx (hunk 2 above)
Comment (max-programming) [reply src/App.tsx:264]: Is it okay if the classnames remain changed if you don't mind? Because finding and changing all of them back again is a pain

File: tsconfig.json (hunk 1 above)
Comment (igitur) [reply tsconfig.json:20]: > Fixed it at [a501aea](https://github.com/cwackerfuss/react-wordle/commit/a501aea421d35245e4644611680511c43a2e27f7)

Please add the commit to this repo.

Comment (igitur) [reply tsconfig.json:20]: > > Fixed it at [a501aea](https://github.com/cwackerfuss/react-wordle/commit/a501aea421d35245e4644611680511c43a2e27f7)
> 
> Please add the commit to this PR.

Sorry, I see it's already included.

**Comment by max-programming** [comment]
Will look into that

**Comment by igitur** [comment]
@max-programming Any progress on getting this to build upon a clean checkout? I think it's a great contribution. We just need to tie a few loose ends.

**Comment by max-programming** [comment]
Sorry @igitur I had my exams. They got over today and I'll work on it

**Comment by igitur** [comment]
Please keep your priorities straight. Wordle clones are much more important than exams! Joking. :-P

**Comment by max-programming** [comment]
I give up now. With all the tries I have done, I can't find a way to do that.
It would've been easier if we were using [Vite](http://vitejs.dev/)

File: src/App.tsx (hunk 2 above)
Comment (igitur) [reply src/App.tsx]: I undid these.

File: tsconfig.json (hunk 1 above)
Comment (igitur) [reply tsconfig.json:20]: This doesn't seem to be a problem anymore. Not sure what fixed it.

**Comment by igitur** [comment]
@max-programming Not sure if you stepped away from this project for good. I took a look at this PR, rebased it on the latest version of `main`, upgraded the new plugin and now all the tests pass. Are you happy with it? if so, I'll merge.

**Comment by max-programming** [comment]
> @max-programming Not sure if you stepped away from this project for good. I took a look at this PR, rebased it on the latest version of `main`, upgraded the new plugin and now all the tests pass. Are you happy with it? if so, I'll merge.

Sure! If everything works alright! You can merge it

**Comment by igitur** [comment]
Thanks!
